debian/intelmq.install

```diff
@@ -1,2 +1,3 @@
-CHANGELOG.md LICENSE COPYRIGHT MANIFEST.in REQUIREMENTS docs/*md usr/share/doc/intelmq/
```

Comment:
The LICENSE file is removed here, but not added (in this bunch of commits). Is it still shipped?